### PR TITLE
Create radio button component

### PR DESCRIPTION
**Pull request by zach-jones-slalom**
image

This PR adds a radio group comp to lakefront.

Still need to write unit tests.

### CODE REVIEW

**Review by michaelpuzon — APPROVED** [review]

File: src/RadioGroup/RadioGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            </StyledLabel>
+          )
+        })
+      }
+      </ThemeProvider>
+    );
+  };
+  
+  export default RadioGroup;
+
```
Comment (jason-ransom-slalom) [inline src/RadioGroup/RadioGroup.tsx:88]: Add a new line at the end of the file.

File: src/RadioGroup/assets/radioChecked.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0z" fill="none"/><path d="M12 7c-2.76 0-5 2.24-5 5s2.24 5 5 5 5-2.24 5-5-2.24-5-5-5zm0-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"/></svg>
```
Comment (jason-ransom-slalom) [inline src/RadioGroup/assets/radioChecked.svg:1]: Add a new line at the end of the file.

File: src/RadioGroup/assets/radioUnchecked.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0z" fill="none"/><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"/></svg>
```
Comment (jason-ransom-slalom) [inline src/RadioGroup/assets/radioUnchecked.svg:1]: Add a new line at the end of the file.

**Review by jason-ransom-slalom — CHANGES_REQUESTED** [review]
You'll want to export and add the new comment to the documentation. [See these steps](https://github.com/ToyotaResearchInstitute/lakefront/blob/main/CONTRIBUTING.md#adding-new-components)

File: src/RadioGroup/RadioGroup.tsx (hunk 1 above)
Comment (jason-ransom-slalom) [reply src/RadioGroup/RadioGroup.tsx:88]: Oh! And @zach-jones-slalom, the branch name should be renamed.

**Comment by jason-ransom-slalom** [comment]
Completing this PR will close #48

**Pull request closed without merging**